plugins/org.eclipse.elk.alg.force/src/org/eclipse/elk/alg/force/Force.melk

```diff
@@ -46,6 +46,12 @@ algorithm force(ForceLayoutProvider) {
     supports org.eclipse.elk.padding = new ElkPadding(50)
     supports org.eclipse.elk.interactive
     supports org.eclipse.elk.portConstraints
+    // Common node micro layout
+    supports org.eclipse.elk.omitNodeMicroLayout
+    supports org.eclipse.elk.nodeSize.options
+    supports org.eclipse.elk.nodeSize.constraints
+    supports org.eclipse.elk.nodeLabels.placement
+    supports org.eclipse.elk.portLabels.placement
```

Comment:
Are these the only options the node placement code uses?

Edit: For example, the stress layouter also support minimum node size?

Reply:
I must have missed the minimum size here - apart from that it's a best guess.